proc/variables.go

```diff
@@ -37,6 +37,22 @@ type M struct {
 	curg     uintptr // Current G running on this thread.
 }
 
+const (
+	// From: src/runtime/runtime2.go
```

Comment:
Condense this comment to `G status, from: ...`.

Also, let's get rid of the `_` prefix, I don't think it is necessary to use the exact same constants from the Go source.